x-pack/plugins/fleet/server/services/epm/archive/save_to_es.ts

```diff
@@ -119,3 +124,35 @@ export async function archiveEntryToBulkCreateObject(opts: {
     attributes: doc,
   };
 }
+
+export const getEsPackage = async (
+  pkgName: string,
+  pkgVersion: string,
+  packageAssets: SavedObjectsBulkGetObject[],
+  savedObjectsClient: SavedObjectsClientContract
+) => {
+  const soRes = await savedObjectsClient.bulkGet<PackageAsset>(packageAssets);
```

Comment:
I think we should add `fields: ['asset_path']` to `packageAssets` before we send to bulkGet. That way it won't return the asset bodies as well. For a large package like aws it's the difference between a few kB and a few MB.

https://www.elastic.co/guide/en/kibana/master/saved-objects-api-bulk-get.html#saved-objects-api-bulk-get-request-body